doc/.netlify/state.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
What does this file do?

Reply:
This is a link to a local site from my machine to the one on Netlify. It was created automatically when i deployed the docs manually from netlify CLI